constellations/gemini-admin/src/classes/classEdit.js

```diff
@@ -99,6 +99,10 @@ export class ClassEditPage extends React.Component {
                         sessions: sessions,
 
                         fullState: classObj.fullState,
+                        googleClassCode: classObj.googleClassCode || "",
+                        priceLump: classObj.priceLump || "",
+                        pricePerSession: classObj.pricePerSession || "",
```

Comment:
priceLump and pricePerSession are usually integers, so you can use: `classObj.priceLump || 0`, which will default to 0 if not available.